core/shared/src/main/scala/scalaz/zio/internal/ExecutionMetrics.scala

```diff
@@ -16,7 +16,7 @@
 
 package scalaz.zio.internal
 
-trait ExecutionMetrics {
+private[internal] trait ExecutionMetrics {
```

Comment:
This trait isn't exposed anywhere???

Reply:
The only place where you might need to implement it - in `Executor`.
But you're right, I've missed it - will revert.